src/styles/main.scss

```
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import "./utils/reset";
+@import "./utils/fonts";
+@import "./utils/variables";
+@import "./utils/width";
+@import "./blocks/header.scss";
```

```suggestion
@import "./utils/width";

@import "./blocks/header.scss";
```
blank line between utils and blocks